File: Module 1 - Decision Making/Decision Making/Switch&match-case/ChallengeProblem/Calculator.cpp
// Calculator 

#include <iostream>
using namespace std;

int main()
{
    float Num1,Num2;
    char Operator;
    cout << "Enter Number: ";
    cin >> Num1;
    cout << "Enter Number: ";
    cin >> Num2;
    cout << "Enter Operator[+,-,*,/,%]: ";
    cin >> Operator;

    switch(Operator)
    {
        case '+':
            cout << Num1+Num2;
            break;

        case '-':
            cout << Num1+Num2;
            break;

        case '*':
            cout << Num1+Num2;
            break;

        case '/':
            cout << Num1+Num2;
            break;

        case '%':
            cout << Num1+Num2;
            break;

        default:
            cout << "Invalid Operator";
    }

    return 0;
}

File: Module 1 - Decision Making/Decision Making/Switch&match-case/ChallengeProblem/Calculator.py
# Calculator 

Num1 = float(input("Enter Number: "))
Num2 = float(input("Enter Number: "))
operator = input("Enter Operator[+,-,*,/,%]: ")

match operator:
    case "+":
        print(Num1+Num2)
    case "-":
        print(Num1-Num2)
    case "*":
        print(Num1*Num2)
    case "/":
        print(Num1/Num2)
    case "%":
        print(Num1%Num2)
    case _:
        print("Invalid Operator")

